nginx/datadog_checks/nginx/data/conf.yaml.example

```diff
@@ -78,7 +78,7 @@ instances:
     # use_plus_api_stream: false
 
     ## @param plus_api_version - integer - optional - default: 2
-    ## Specify the version of the Plus API to use. The check supports versions 1 & 2.
+    ## Specify the version of the Plus API to use. The check supports versions 1-3.
```

Comment:
This file is generated from `nginx/assets/configuration/spec.yaml`: https://datadoghq.dev/integrations-core/meta/config-specs/

You'd need to make changes to that file first, then run `ddev validate config -s` to sync this `conf.yaml.example` file. :-)

That's what CI is pointing at, at least:

https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=41544&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=57642e18-890a-5eef-b695-2b3c448c1261&l=16

```
Validating default configuration files...
nginx:
    File `conf.yaml.example` is not in sync, run "ddev validate config -s"
```